build.gradle

```diff
@@ -8,6 +8,7 @@ buildscript {
         lifecycle_version = "2.5.0-rc01"
         coroutine_version = "1.6.1"
         timber_version = '5.0.1'
+        coil_version = "2.1.0"
```

Comment:
Can we use glide instead of coil, it looks to be faster.
https://proandroiddev.com/coil-vs-picasso-vs-glide-get-ready-go-774add8cfd40

Reply:
Also if you look at the repos it has a better following.
https://github.com/bumptech/glide
https://github.com/coil-kt/coil